android/walletlib/src/main/java/com/solana/mobilewalletadapter/walletlib/authorization/AuthorizationsDao.java

```diff
@@ -0,0 +1,132 @@
+/*
+ * Copyright (c) 2022 Solana Mobile Inc.
+ */
+
+package com.solana.mobilewalletadapter.walletlib.authorization;
+
+import android.content.ContentValues;
+import android.database.Cursor;
+import android.database.sqlite.SQLiteDatabase;
+import android.database.sqlite.SQLiteStatement;
+import android.net.Uri;
+
+import androidx.annotation.NonNull;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/*package*/ class AuthorizationsDao extends DbContentProvider<AuthRecord>
+        implements AuthorizationsSchema, PublicKeysSchema,
+        WalletUriBaseSchema, AuthorizationsDaoInterface {
+
+    AuthorizationsDao(SQLiteDatabase db) {
+        super(db);
+    }
+
+    @Override
+    protected AuthRecord cursorToEntity(Cursor cursor) {
+        return null;
```

Comment:
If this isn't ever intended to be used, I'd throw an `UnsupportedOperationException` here, to make sure we fail fast due to presumably a programming error.

Reply:
Done.